controller/internal/routes/config_api_model.go

```diff
@@ -125,27 +125,47 @@ func MapConfigToRestModel(ae *env.AppEnv, config *model.Config) (*rest_model.Con
 	return ret, nil
 }
 
```

Comment:
This is a style thing, so feel free to ignore, but I would prefer `func narrowJsonType(v interface{}) interface{}`